x-pack/test/detection_engine_api_integration/security_and_spaces/tests/runtime.ts

```diff
@@ -97,7 +99,16 @@ export default ({ getService }: FtrProviderContext) => {
         await waitForRuleSuccessOrStatus(supertest, id);
         await waitForSignalsToBePresent(supertest, 4, [id]);
         const signalsOpen = await getSignalsById(supertest, id);
-        const hits = signalsOpen.hits.hits.map((signal) => signal._source?.host);
+        const hits = signalsOpen.hits.hits
+          .map((signal) => signal._source?.host as Array<{ name: string }>)
+          .map((host) => {
+            // sort the inner array elements first
+            return host.sort((a, b) => a.name.localeCompare(b.name));
+          })
+          .sort((aArray, bArray) => {
+            // since these are all unique, using just the first element should give us stability
```

Comment:
If this starts failing would that point to some sort of issues with dedupe? (Seeing as we have had some with all the recent big changes)

Reply:
It would be an issue with de-dupes and not the sort order but that ticket this fixes is definitely sort order issue.